account/manager.go

```diff
@@ -1546,19 +1561,12 @@ func (m *manager) spendAccount(ctx context.Context, account *Account,
 	// With the transaction crafted, update our on-disk state and broadcast
 	// the transaction. We'll need some additional modifiers if the account
 	// is being modified.
-	prevAccountState := account.Copy()
 	if err := m.cfg.Store.UpdateAccount(account, modifiers...); err != nil {
 		return nil, nil, err
 	}
 
-	// As this is a generic account modification, we'll add some additional
-	// information to make accounting for this transaction a bit easier.
-	deposit := prevAccountState.Value < account.Value
 	acctKey := account.TraderKey.PubKey.SerializeCompressed()
-	contextLabel := fmt.Sprintf(" poold -- AccountModification(acct_key=%x, "+
-		"expiry=%v, deposit=%v, is_close=%v)", acctKey,
```

Comment:
i think we should leave the `expiry` tag here.

For [legacy](https://github.com/lightninglabs/pool/blob/master/poolscript/script.go#L58) and [taproot](https://github.com/lightninglabs/pool/blob/master/poolscript/script.go#L93) accounts the expiry date is part of the scrip. That means that the user needs that parameter to recreate the spending script.

If some day our server goes down, the client-side only recovery  account process could have a nice speed up by using this data, instead of having to perform a brute force (as we have right now).

Reply:
That's a good point, thank you!

Currently the expiry tag is just a bool. Which is never set to true as far as I can tell. If I understand you correctly, perhaps it should be a date?